### ISSUE TITLE
Credential Endpoint Proposal

### ISSUE THREAD

**Issue by tplooker** [mattrglobal/oidc-client-bound-assertions-spec#27]
@kimcameron has made a proposal for the credential endpoint documented [here](https://hackmd.io/My20MxGJQqGgVVg7CLJFhQ)

**Comment by AdamJLemmon** [comment]
@kimcameron @tplooker as per above PR I added an initial minimal draft for review.

I drafted using the signed request pattern as the proof of control of the `sub` claim within the request.  
I took this approach as opposed to the `credential_sub` and `credential_sub_dpop` above as I was guessing that the `credential_sub_dpop` would likely end up looking fairly similar to the signed request.
However, we could likely use the signed request as the `credential_sub_dpop` and pull it's sub to `credential_sub` if that feels better. Same goes for `credential_format` if we wanted to pull to a top level request property.

Kindly let me know what you feel best!

I have also left out dpop for now to get this first draft merged in and then was thinking to determine how we wanted to introduce dpop or related sender constraints / authZ etc..  As that will require a little rework upstream of the credential request as well.
